client/spa/js/learning-resource/spec/learning-resource.view.spec.js

```diff
@@ -0,0 +1,138 @@
+/*
+global jasmine, describe, it, expect, beforeEach, afterEach, xdescribe, xit,
+spyOn
+*/
+
+// Get the code you want to test
+var View = require('../learning-resource.view.js');
+var matchers = require('jasmine-jquery-matchers');
+var Backbone = require('../../vendor/index').Backbone;
+
+// Test suite
+console.log('test learning-resource.view');
+describe('Learning resource view ', function(){
+
+  var model;
+  var view;
+
+  beforeEach(function(){
+    var Model = Backbone.Model.extend({});
+
+    // Add some convenience tests for working with the DOM
+    // Helper for toHaveClass
+    jasmine.addMatchers(matchers);
+
+    // Needs to have the fields required by the template
+    model = new Model({
+      title: 'Crisis Averted',
+      resourceType: 'presentation',
+      description: 'You are welcome.',
+      authors: ['Mary', 'Joe']
+    });
+
+    view = new View({
+      model: model
+    });
+  });
+
+  describe('when the view is instantiated ', function(){
+
+    it('creates the correct element', function(){
+      // Element has to be uppercase
+      expect(view.el.nodeName).toEqual('DIV');
+    });
+
+    it('sets the correct class', function(){
+      expect(view.$el).toHaveClass('learning-resource');
+    });
+
+  });
+
+  describe('when the view is rendered ', function(){
+
+    it('returns the view object ', function(){
+      expect(view.render()).toEqual(view);
+    });
+
+    it('produces the correct HTML ', function(){
+      view.render();
+      expect(view.$('#title').val()).toEqual('Crisis Averted');
+    });
+
+  });
+
+  describe('when the user clicks on the Edit button ', function(){
+
+    beforeEach(function(){
+      view.$('button .b-edit').trigger('click');
+      view.render();
+    });
+
+    describe('and when the user inputs new information ', function(){
+
+      beforeEach(function(){
```

Comment:
Here I'm trying to simulate the user inputting new information. When I run the test it picks up the initial information defined in the model instead. 

Here is my error:<pre><code>Expected 'Crisis Averted' to equal 'changed title'.
            at Object.<anonymous> (C:/Users/JSTADR~1/AppData/Local/Temp/25ac7c575ff07e6c139cf69ca01edbec55aab123.browserify:425:42 <- client\spa\js\learning-resource\spec\learning-resource.view.spec.js:105:0)
        Expected 'You are welcome.' to equal 'changed description'.
            at Object.<anonymous> (C:/Users/JSTADR~1/AppData/Local/Temp/25ac7c575ff07e6c139cf69ca01edbec55aab123.browserify:426:41 <- client\spa\js\learning-resource\spec\learning-resource.view.spec.js:106:0)
        Expected 'Mary, Joe' to contain 'sis'.
            at Object.<anonymous> (C:/Users/JSTADR~1/AppData/Local/Temp/25ac7c575ff07e6c139cf69ca01edbec55aab123.browserify:427:41 <- client\spa\js\learning-resource\spec\learning-resource.view.spec.js:107:0)
        Expected 'presentation' to equal 'link'.
            at Object.<anonymous> (C:/Users/JSTADR~1/AppData/Local/Temp/25ac7c575ff07e6c139cf69ca01edbec55aab123.browserify:428:65 <- client\spa\js\learning-resource\spec\learning-resource.view.spec.js:108:0)
LOG: 'Mary, Joe'
LOG: 'Mary, Joe'</code></pre>